R/glmnot.R

```diff
@@ -121,14 +121,27 @@ glmnot.default <- function(X, y, family,
                            pmax = +Inf,
                            formula = NULL,
                            xlev = NULL,
-                           weights = rep(1, nrow(X))) {
-  m <- cv.glmnet(X, y, family = family,
-              alpha = alpha,
-              type.measure = type.measure,
-              pmax = pmax,
-              nfolds = nfolds,
-              weights = weights,
-              parallel = TRUE)
+                           weights = rep(1, nrow(X)),
+                           intercept = TRUE
```

Comment:
Perhaps it makes more sense to always set intercept = FALSE, and then let the user control the intercept using the formula?